hi/codereview.py

```diff
@@ -53,12 +53,6 @@ def fetch_added_lines_only(file):
     added_lines = [line for line in patch.splitlines() if line.startswith('+') and not line.startswith('+++')]
```

Comment:
There are a couple of critical issues related to proper input validation and the absence of hard-coded secrets. Additionally, there seems to be a lack of structure in the way instructions are formatted.

```
+                            "Please review the code changes provided in the diff below based on the following criteria:\n\n"
+                            "\n\nIf the overall code appears to be 80% good or more and has no critical issues, respond with: 'Everything looks good.'"
+                            " If there are critical issues that need attention, provide a brief summary (max 2 sentences) of the key areas needing improvement."
+                            " Include a code snippet from the diff that illustrates the issue, without suggesting detailed solutions or minor improvements."
+                            "\n\nKeep the response brief, as if it were from a human reviewer."
```